src/pages/AddCredentials/AddCredentials.js

```diff
@@ -150,14 +150,14 @@ const Issuers = () => {
 						style={{ maxHeight: '80vh' }}
 					>
 						{filteredIssuers.map((issuer) => (
-							<li
+							<button
```

Comment:
These `<button>`s are now children of a `<ul>`, perhaps that should be changed to `<div>` or something like that instead?